tests/test_identifiers.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+
+import httpretty
+from factory import SubFactory
+from nose.tools import *  # noqa
```

Comment:
For future reference: new versions of flake8 are stricter with this pragma: you now need to do ` # flake8: noqa`